src/SFA.DAS.DATA.MANAGEMENT/Tables/ASDataPL_Assessor_Certificates.sql

```diff
@@ -0,0 +1,39 @@
+﻿Create Table [ASData_PL].[Assessor_Certificates]
+(
+[Id]											[uniqueidentifier]		PRIMARY KEY NOT NULL,
+[AchievementDate]								[DateTime2](7)			NULL,
+[BatchNumber]								    [Int]					NULL,
+[CertificateReference]							[NVarchar](50)			NULL,
+[CertificateReferenceId]						[Int]					NULL,
+[ContactName]									[NVarchar](100)			NULL,
+[ContactOrganisation]							[NVarchar](200)			NULL,
+[ContactPostCode]								[NVarchar](20)			NULL,
+[CourseOption]									[NVarchar](200)			NULL,
+[CreatedAt]										[DateTime2](7)			NULL,
+[CreateDay]										[Date]					NULL,	
+[CreatedBy]										[NVarchar](250)			NULL,
+[DeletedAt]										[DateTime2](7)			NULL,
+[DeletedBy]										[NVarchar](250)			NULL,
+[Department]									[NVarchar](200)			NULL,
+[IsPrivatelyFunded]								[Bit]					NULL,
+[LearnerFamilyName]								[NVarchar](200)			NULL,
+[LearnerGivenNames]								[Nvarchar](100)			NULL,
```

Comment:
Might be good to exclude if there is no Business Requirement

Reply:
As discussed taken off all the columns which are not required , mainly created by, updated by, deleted by are removed and no PII data in the left of the columns of the schema